### PR TITLE
UI: Base Layout

### PR DESCRIPTION
**Pull request by goldsteinsveta**
#47 fixed

Screenshots, highlighted:
Screenshot 2021-04-23 at 17 53 10Screenshot 2021-04-23 at 17 53 21

### CODE REVIEW

File: src/components/theme.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```
Comment (hbulgarini) [inline src/components/theme.tsx:1]: Is this file necessary to be created as `.tsx` ? i feel this can be a `.ts` file which can live in a different folder than components. For example: `./src/styles` ? In case you want to leave it as a component, please rename the file as `Theme.tsx`

File: src/screens/Main.tsx [hunk 2]
```diff
@@ -39,50 +40,52 @@ export function Main({ className }: Props) {
 
   return (
     <>
```
Comment (hbulgarini) [inline src/screens/Main.tsx:42]: we can remove this since `BoxMain` enclose all

**Review by hbulgarini — CHANGES_REQUESTED** [review]

**Comment by hbulgarini** [comment]
@goldsteinsveta it seems we have some conflicts with App / Maint.tsx please let mw know if you need help for solving them.

File: src/components/theme.tsx (hunk 1 above)
Comment (wirednkod) [reply src/components/theme.tsx:1]: Not really agree with that. All files that contain JSX elements should be TSX while pure Typescript files should be TS.

**Review by wirednkod — APPROVED** [review]
I agree with @hbulgarini on the removal of <></>. Other than that PR lgtm

File: src/components/theme.tsx (hunk 1 above)
Comment (hbulgarini) [reply src/components/theme.tsx:1]: right, this is why i asked first to rename it to ".ts" . Anyway i left the chance to rename it as component because i was not sure if sveta was planning to introduce JSX code in it. In case she does not, then rename it to ".ts" is the right thing to do.

Comment (wirednkod) [reply src/components/theme.tsx:1]: yeap - my mistake

File: src/screens/Main.tsx (hunk 2 above)
Comment (goldsteinsveta) [reply src/screens/Main.tsx]: done

File: src/components/theme.tsx (hunk 1 above)
Comment (goldsteinsveta) [reply src/components/theme.tsx]: done, it's .ts now

**Review by wirednkod — APPROVED** [review]

**Review by hbulgarini — APPROVED** [review]
LGTM!
